guide/how-it-works/ecash/overview.md

```diff
@@ -0,0 +1,222 @@
+
+### Creating and redeeming ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   alt-text = "An illustration of the process of creating and redeeming ecash tokens"
+   caption = "Creating and redeeming ecash tokens."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+The process of creating ecash on Bitcoin is known as **minting.** The process of redeeming ecash into sats via the Lightning Network is known as **melting.** This process typically includes the following steps:
+
+#### Minting ecash
+
+**1. User Deposits Funds** - The user deposits Bitcoin into a mint. Typically this is done by generating a LN invoice through the mint and paying it.
+
+**2. Blinded Token Creation** - Upon successful payment, the mint generates blinded tokens, which are cryptographic representations of the deposited funds. The blinding process ensures that the mint cannot link the tokens to the user, preserving privacy.
+
+**3. Mint provides Tokens to User** - The user receives a set of tokens that can be combined to create an ecash token of any amount denominated in Bitcoin.
+
+#### Melting ecash
+
+**4. User Initiates Melting** - The user requests to redeem their ecash tokens for Bitcoin through the Lightning Network.
+
+**5. Token destruction** - The mint verifies and destroys the ecash tokens, ensuring they cannot be reused. This step confirms the authenticity of the tokens and maintains the integrity of the system.
+
+**6. Sats Transfer via Lightning Network** - The user receives an equivalent amount of Bitcoin, completing the melting process and converting ecash back into spendable Bitcoin.
+
+### Sending and receiving ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   alt-text = "An illustration of the process of sending and receiving ecash tokens to various users"
+   caption = "Sending and receiving ecash tokens."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+Sending ecash and receiving can be done via multiple methods to suit different transaction needs. One of the advantages of ecash is that users are simply transacting pieces of data. These pieces of data are very flexible and can be transmitted via any medium or rail. Some unique ways that ecash can be sent and received, which are not possible with on-chain or Lightning transactions, include:
```

Comment:
> simply transacting pieces of data

I'd like more clarity on that. On the base layer, you share an address, on lightning an invoice, which are also pieces of data. How is this different? What is contained in this piece of data, and what ensures that the sender can't double-spend it?

Reply:
What I’m trying to communicate here is that the data is the money—it’s the ecash token, the bearer asset. Simply calling it “data” might be too general. I could explain in more detail that with ecash, the user is transmitting strings of text, which are the bearer tokens representing an IOU for sats.

Update:
"Sending ecash and receiving can be done via multiple methods to suit different transaction needs. One of the advantages of bitcoin-backed ecash is that the tokens are just strings of text, which are very flexible and can be transmitted via any medium or rail. Some unique ways that ecash can be sent and received, which are not possible with on-chain or Lightning transactions, include:"

thoughts?

Reply:
I was trying to roughly visualize this:

![image](https://github.com/user-attachments/assets/16da4c48-8329-40e1-899c-242c95ccec1e)

So with bitcoin and lightning, Alice needs to share something with Bob, a transactions gets created that is routed by a network and then arrives in Alice's wallet.

With ecash, Bob gets the ecash token from the mint and gives it directly to Alice, who then verifies it with the mint. No addresses or invoices, and no network routing.

Is that right?

There's probably some way to use a text-diagram combo to make this point super clear.

Reply:
Yeah, you got it! I like this skeleton, it's good to see ecash token sending and in direct comparison with base layer and ln. I'll work on an illustration that captured this.

Reply:
The paper analogy works well to explain how the ecash bearer tokens maintain privacy properties, but once you examine how it actually functions it falls apart because ecash is fundamentally digital and doesn’t operate like paper cash in many ways. Unlike paper cash, with ecash, the token (or bill) is destroyed and recreated every time it’s used. This process is key because it prevents double spending. If we want to explain this, then i think the paper cash analogy fit well at this level.

Reply:
Added this illustration. It's now in the introduction page as per @mouxdesign suggestion (https://deploy-preview-1093--bitcoin-design-site.netlify.app/guide/how-it-works/ecash/introduction/).